xsnap/sources/xsnap.c

```diff
@@ -36,37 +36,33 @@ extern void xs_base64_encode(xsMachine *the);
 extern void xs_base64_decode(xsMachine *the);
 extern void modInstallBase64(xsMachine *the);
 
-#define mxSnapshotCallbackCount 23
+#define mxSnapshotCallbackCount 21
 xsCallback gxSnapshotCallbacks[mxSnapshotCallbackCount] = {
 	xs_issueCommand, // 0
 	xs_print, // 1
-	xs_gc, // 2
-
-	xs_clearTimer, // 3
```

Comment:
clearTimer is doing triple-duty as for both kinds of timers and interval. I’m sure Agoric doesn’t need clearInterval, so this should be fine, and implies that we can comment out xs_clearTimer.